lib/node/nodes/contour-map.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'contour';
+
+var PARAMS = {
+    // the scattered points dataset with real values
+    source: Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    // column of values in source dataset
+    column: Node.PARAM.STRING(),
+    // buffer around my dataset
+    buffer: Node.PARAM.NULLABLE(Node.PARAM.NUMBER(), 0.2),
+    // 0: nearest neighbor, 1: barymetric, 2: IDW
+    method: Node.PARAM.NULLABLE(Node.PARAM.ENUM('nearest_neighbor', 'barymetric', 'IDW'), 'barymetric'),
+    // classification method
+    class_method: Node.PARAM.NULLABLE(Node.PARAM.ENUM('equals', 'heads_and_tails', 'jenks', 'quantiles'), 'quantiles'),
```

Comment:
I would rename head_and_tails to headstails to be coherent with cartodb-postgresql extension methods.